tests/suites/hup/suite.js

```diff
@@ -154,20 +154,26 @@ const initDUT = async (that, test, target) => {
 	await that.worker.addSSHKey(that.sshKeyPath);
 
 	// create tunnels
-	console.log('Creating SSH tunnels to DUT');
-	await that.worker.createSSHTunnels(
-		that.link,
-	);
+	test.comment(`Creating SSH tunnels to DUT - trying to find IP of ${that.link}`);
+	await test.resolves(
+		this.worker.createSSHTunnels(
+			that.link,
+		),
+		`Should detect ${this.link} on local network and establish tunnel`
+	)
 
-	test.comment(`Waiting for DUT to be reachable`);
-	await that.utils.waitUntil(async () => {
-		return (
-			(await that.worker.executeCommandInHostOS(
-				'[[ -f /etc/hostname ]] && echo pass || echo fail',
-				target,
-			)) === 'pass'
-		);
-	}, true);
+	await test.resolves(
+		that.utils.waitUntil(async () => {
+			test.comment(`Trying to SSH into device ${target}`);
+			return (
+				(await that.worker.executeCommandInHostOS(
+					'[[ -f /etc/hostname ]] && echo pass || echo fail',
+					target,
+				)) === 'pass'
```

Comment:
I think this would be clearer with a `.then()` instead of comparing an awaited promise with a string.

```
return this.worker.executeCommandInHostOS(...).then(res => res === 'pass')
```